chart/keel/templates/deployment.yaml

```diff
@@ -28,7 +28,7 @@ spec:
       containers:
         - name: keel
           # Note that we use appVersion to get images tag.
-          image: "{{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}"
+          image: "{{ .Values.image.repository }}:{{ .Chart.AppVersion | default .Values.image.tag }}"
```

Comment:
You are introducing a regression here : `image.tag` defined by user in `values.yaml` will not be used anymore.

Reply:
Should be fixed now.  I also commented out the keel config from the values.yaml file, since it wouldn't actually update because deployment.yaml isn't even using `image.tag`.